azure_functions_worker/bindings/generic.py

```diff
@@ -34,6 +34,8 @@ def encode(cls, obj: Any, *,
 
     @classmethod
     def decode(cls, data: datumdef.Datum, *, trigger_metadata) -> typing.Any:
+        if data is None:
```

Comment:
Adding a comment would help on why data is None is a valid case now. Or just reference to the issue would help